### PR TITLE
fix: don't change buttons order in actionbar "More" popover

### PR DESCRIPTION
**Pull request by vkurlyan**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: coral-component-actionbar/examples/index.html [hunk 1]
```diff
@@ -31,12 +31,12 @@
       });
     </script>
 
-    <link rel="stylesheet" href="../dist/css/coral.css">
```
Comment (CezCz) [inline coral-component-actionbar/examples/index.html:34]: Why is this change necessary ? 
It seems to be that coral.css is downloaded correctly here
https://opensource.adobe.com/coral-spectrum/examples/#actionbar

File: coral-component-actionbar/src/scripts/ActionBarSecondary.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ class ActionBarSecondary extends ActionBarContainer(BaseComponent(HTMLElement))
     let item = null;
     let wrappedItem = null;
 
-    for (let i = 0 ; i < this._itemsInPopover.length ; i++) {
```
Comment (CezCz) [inline coral-component-actionbar/src/scripts/ActionBarSecondary.js:32]: I am not sure how this solves the problem of ordering, can you provide more insights?

**Comment by CezCz** [comment]
Please remove CQ-4317286 from the commit message and PR title. It should start with "fix".

File: coral-component-actionbar/src/scripts/ActionBarSecondary.js (hunk 2 above)
Comment (vkurlyan) [reply coral-component-actionbar/src/scripts/ActionBarSecondary.js:32]: Hi @CezCz, in this function, buttons are returned from popover to <coral-actionbar-secondary>. They are added after the "More button" one by one (line 43). It should be done in reversed order to keep buttons order which was before buttons were added to the popover.

File: coral-component-actionbar/examples/index.html (hunk 1 above)
Comment (vkurlyan) [reply coral-component-actionbar/examples/index.html]: Reverted

File: coral-component-actionbar/src/scripts/ActionBarSecondary.js (hunk 2 above)
Comment (CezCz) [reply coral-component-actionbar/src/scripts/ActionBarSecondary.js:32]: I see, but can you provide an example? I am trying to reproduce this issue on the coral-spectrum playground but I am not sure on how to follow. In the examples it seems the order is correct. I checked the internal jira tickets and it's not clear for me there either. 

This change might seem innocent, however if there are clients depending on the reverse ordering this will cause a regression on their side. Did you consider this? How do you think we should proceed here?

Comment (vkurlyan) [reply coral-component-actionbar/src/scripts/ActionBarSecondary.js:32]: @CezCz , 
**Steps to reproduce:**
1. Open https://opensource.adobe.com/coral-spectrum/examples/#actionbar
2. In second example, there is "more" button on right side. Click on it to open popover. "Search" button is on the first place in the popover. 
3. Close the popover and open it again.
**Actual behaviour:** "Search" button is on the last place. Buttons order is reversed.
**Expected behaviour:** "Search" button is still on the first place in the popover.

It also may cause another issue, if user has hidden buttons under "more" button, opens and closes "More" popover, then increases browser width to display hidden buttons from popover. Their order will be reversed.

I've checked "Coral/coralui-component-actionbar" repo. It's already fixed there in the same way as suggested in this PR. 
I think we should fix it as it's very unlikely that clients are dependent on changing buttons order and if somebody is then he might lock coral-spectrum version in package.json until fix it.
